app/controllers/api/v1/gifts_controller.rb

```diff
@@ -1,11 +1,39 @@
 class Api::V1::GiftsController < ApplicationController
+  before_action :find_friend
+  before_action :find_gift, only: %i[show destroy]
+
   def index
-    @gifts = Gift.all
-    render json: @gifts
+    render json: FriendSerializer.new(@friend, include: [:gifts])
```

Comment:
This makes sense, wonder if we might have a better AR query we could use but nothing comes to mind and this works!